rules/style.css

```
@@ -137,18 +137,35 @@ nav.TOC.main {
   }
 }
 
-
 /* *** Colors *** */
 
 :root {
+  color-scheme: light dark; /* Default supports both light and dark schemes */
+
+  /* Light mode default variables */
   --main-background-color: #ccc5b3;
   --accent-background-color: #313831;
   --text-background-color: #ffffff;
   --text-color: #000000;
 }
 
-:root {
-  color-scheme: light;
+@media (prefers-color-scheme: light) {
+  :root {
+    /* Light theme styles */
+    --main-background-color: #ccc5b3;
+    --accent-background-color: #313831;
+    --text-background-color: #ffffff;
+    --text-color: #000000;
+  }
+}
+
+@media (prefers-color-scheme: dark) {
+  :root {
+    --main-background-color: #4d4533;
```

```suggestion
    --main-background-color: #313831;
```
heck, this could work too?

---

addressed in the latest commit